TaskModules/powershell/TaskModuleSqlUtility/JsonConvertionUtility.ps1

```diff
@@ -0,0 +1,63 @@
+function ConvertFrom-Json20([object] $InputObject){ 
```

Comment:
Why these intermediate functions, why can't the logic go to else part of the code?